src/test/java/PageFactory/BasePage.java

```diff
@@ -0,0 +1,57 @@
+package PageFactory;
+
+import POM.pages.AllSongsPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePage {
+    WebDriver driver;
+    WebDriverWait wait;
+    Actions actions;
+
+    @FindBy(how = How.CSS,  using = "img.avatar")
+     WebElement avatarLocator;
+
+   @FindBy (how = How.CSS, using ="[data-testid = 'sound-bar-play']")
+   WebElement soundBarPlayLocator;
+
+
+    @FindBy (how = How.CSS, using = "li a.songs" )
+    WebElement allSongsMenuItemLocator;
```

Comment:
**I recommend:**  make all the locators or elements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example



```
  @FindBy(how = How.CSS, using = "img.avatar")
  private WebElement avatarLocator;

  @FindBy(how = How.CSS, using = "[data-testid = 'sound-bar-play']")
  private WebElement soundBarPlayLocator;

  @FindBy(how = How.CSS, using = "li a.songs")
  private WebElement allSongsMenuItemLocator;
```